src/shared/directives/conditional-show/conditional-show-form.directive.ts

```diff
@@ -114,9 +114,9 @@ export class ConditionalShowFormDirective implements OnInit, AfterViewInit, OnDe
     this.evaluateControl(c);
   }
 
-  private handleFormArray = (c: FormArray, caseField: CaseField) => {
+  private handleFormArray = (c: FormArray) => {
     this.evaluateControl(c);
-    c.controls.forEach((formControl, ix) => {
+    c.controls.forEach(formControl => {
```

Comment:
Slightly pedantic but I'd suggest typing `formControl` as `FormControl` or, at the very least, `AbstractControl`.

Reply:
Hmmm... I refactored this one to do away with the need for that - i.e., I ditched the unused `ix` parameter and removed the wrapping parentheses. Given `c.controls` is an array of `AbstractControl`, the type is already known at this point, at least as far as TypeScript is concerned. It feels a bit like adding `: AbstractControl` here would be akin to one of those lint errors where it declares it's trivial (`const bob: string = 'bob'; // lint error`).

Reply:
Ah yes, I'd not considered that - in which case, it's fine as is.